tracing/jaeger/model/json/model.go

```diff
@@ -54,6 +54,7 @@ type Trace struct {
 	Spans     []Span                `json:"spans"`
 	Processes map[ProcessID]Process `json:"processes"`
 	Warnings  []string              `json:"warnings"`
+	Matched   int                   `json:"matched"`
```

Comment:
Perhaps the name of this variable is causing my confusion. "Matched" seems like it would be a boolean: something "matched".. but I see this is of type `int` so I think this is referring to the number of matches (??) ... if that is correct, I would consider changing this to be more meaningful... say, "NumMatched" or "MatchCount" or something like that. This would avoid confusion for people unfamiliar with the code and just seeing the name of the variable as "matched".

Reply:
See previous comment